### DISCUSSION TITLE
Suggestions for router configuration

### DISCUSSION THREAD

**Discussion by di5cord20** [tteck/Proxmox#2336] [Help Needed]
I'm currently using a Unifi router and am looking for suggestions to optimize for Proxmox containers.  I currently specify IPs to match container IDs when I install a container (i.e., container 222 would be assigned IP 192.168.94.222).  I reserved 192.168.94.200-255 for containers but surprisingly, am close to running out.  

I've seen others use a 10.x.x.x address for selfhosting.  Is this something I should consider?  Just looking for some good hygiene practices to explore.  Thanks

**Comment by manuelkamp** [comment]
yes, reconsider using a /8 network instead.
for example:
Network devices (router, modem, default gateway, switches): 10.0.0.1-10.0.0.255
Proxmox (and pbs) nodes: 10.0.1.1-10.0.1.255
Proxmox VM and LXCs: 10.1.1.1-10.1.1.255
IP Cameras: 10.1.2.1-10.1.2.255
Smart Devics: 10.1.3.1-10.1.3-255
and so on...
this gives you also the advantage in vlans to see it better when based on ip.

**Reply by di5cord20** [reply]
Thanks @manuelkamp. After configuring unifi, how do I designate which network Proxmox VM and LXCc should connect to?  Also, any suggestions on VLAN setup and rules?  I'm going to google for articles today but thought I'd ask here first.  Thanks in advance.

**Reply by manuelkamp** [reply]
I have set up my proxmox nodes a static ip and LXC configured dhcp. my dhcp server gives the correct (static) ip based on LXCs MAC address. I do not use unifi, so I cannot tell you how exactly to do it there. For VLAN - it really depends on what services you are running and how they are connected (for example central sql server) or need to be connected (for example reverse proxy). There is no general "this is exactly how you do it", but many best-practices you find via web search and find those which are best for your usecase. Happy homelabing :)
